h/traversal/__init__.py

```diff
@@ -2,38 +2,21 @@
     AnnotationContext,
     GroupContext,
     GroupUpsertContext,
-    OrganizationContext,
     UserContext,
 )
+from h.traversal.organization import (
+    OrganizationContext,
+    OrganizationLogoRoot,
+    OrganizationRoot,
+)
 from h.traversal.roots import (
     AnnotationRoot,
     AuthClientRoot,
     BulkAPIRoot,
     GroupRoot,
     GroupUpsertRoot,
-    OrganizationLogoRoot,
-    OrganizationRoot,
     ProfileRoot,
     Root,
     UserRoot,
     UserUserIDRoot,
 )
-
-__all__ = (
-    "Root",
-    "AnnotationRoot",
-    "AuthClientRoot",
-    "BulkAPIRoot",
-    "OrganizationRoot",
-    "OrganizationLogoRoot",
-    "GroupRoot",
-    "GroupUpsertRoot",
-    "ProfileRoot",
-    "UserRoot",
-    "UserUserIDRoot",
-    "AnnotationContext",
-    "OrganizationContext",
-    "GroupContext",
-    "GroupUpsertContext",
-    "UserContext",
-)
```

Comment:
This is causing lint issues for the moment. Adding `# noqa: F401` doesn't appear to help. As we use `flake8` rather than the usual I'm not really sure what to do about it. I could just put this back I suppose?

Reply:
I gave up on this. I can't work out how to disable the warning.